database/migrations/2024_02_28_212635_add_is_public_to_screen_templates_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('screen_templates', function (Blueprint $table) {
+            $table->boolean('is_public', false)->after('manifest');
```

Comment:
Hey @sanjacornelius This code attempts to set a default value for the boolean column directly within the boolean() method, which is not the correct syntax. Laravel's Schema Builder expects the default value to be set using the default() method chained after the column definition.

Would you consider using the following?
`$table->boolean('is_public')->default(false)->after('manifest');`